core/sis-utility/src/main/java/org/apache/sis/util/collection/WeakValueHashMap.java

```diff
@@ -464,6 +463,7 @@ public V put(final K key, final V value) throws NullArgumentException {
      *
      * @since 0.7
      */
+    @Override
```

Comment:
In this particular case, this annotation can not be applied on a branch targeting JDK7.

Reply:
Thanks, going to undo.